php-exercise4/src/control/control.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Please use proper naming convention for your files and classes.

Please refer to PSR-1 and PSR-12 for best practices.

Reply:
Done